java/client/src/main/java/module-info.java

```diff
@@ -10,6 +10,7 @@
     exports glide.api.models.commands.scan;
     exports glide.api.models.commands.stream;
     exports glide.api.models.commands.FT;
+    exports glide.api.models.commands.json;
```

Comment:
json vs JSON

Reply:
don't java pakcages should be lower case?

Reply:
Well... I don't know the guidelines actually. Do you want to ask hivemind?
If needed, please fix `FT`.

Reply:
Will leave as is. FT could have it's own follow up PR if nessesary.